CentralManagement/ChocoCCM Command Reference/InvokeArchiveCCMDeployment.md

```diff
@@ -0,0 +1,91 @@
+# Invoke-ArchiveCCMDeployment
+
+<!-- This documentation is automatically generated from /Invoke-ArchiveCCMDeployment.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+Archive a CCM Deployment
+
+## Syntax
+
+~~~powershell
+Invoke-ArchiveCCMDeployment `
+  [-Deployment <String>] `
+  [-WhatIf] `
+  [-Confirm] [<CommonParameters>]
+~~~
```

Comment:
This command needs to be renamed. `Invoke` isn't the right verb for this. I'd recommend doing this before first release.

Reply:
Renamed to `Disable-CCMDeployment` and made the help doc a bit clearer